test/fixtures/gatherers/os-release.basic

```diff
@@ -0,0 +1,12 @@
+NAME="Ubuntu"
+VERSION="20.04.3 LTS (Focal Fossa)"
+ID=ubuntu
```

Comment:
I have always seen values wrapped in quotes. Does ubuntu have them without?
It would be nicer to put opensuse :see_no_evil:

Reply:
The quotes are not mandatory AFAIK, see some examples: https://manpages.ubuntu.com/manpages/focal/man5/os-release.5.html

I think they are only required when they include values with spaces in them.

I've added some openSUSE love in the mocked data :-)